runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/RegisterTask.java

```diff
@@ -296,7 +258,8 @@ else if ("http".equals(configURL.getProtocol()) || "https".equals(configURL.getP
             }
             future.join();
 
-            extensions.forEach(e -> e.onConfigured(context));
+            extensions.forEach(e -> e.onRegistered(context));
+            return namespace;
```

Comment:
Define `NamespaceConfig namespace = null;` at top of method, and use single return at end of method.

At the moment, it looks like you might be skipping over the error reporting after the try/catch.